ProjectSubmit/Gemfile

```diff
@@ -41,6 +41,19 @@ group :development do
   gem 'web-console'
 end
 
+<<<<<<< HEAD
```

Comment:
Isto marca o início de um conflito por resolver. O bloco é definido por
```
<<<<<< HEAD
código do master em possível conflito
=======
código do nosso branch (neste caso origin/caria) em possível conflito.
>>>>>> origin/caria
```

 É preciso analisar os dois blocos e ver o que faz sentido manter. Se houver dúvidas o ideal é falar / pedir opinião ou ajuda a mais alguém, idealmente a pessoa que fez a alteração anterior. Neste caso a ideia parece-me só manter a gem nova que foi adicionada pelas tags do @VascoJM (acts-as-taggable-on) e manter as novas gems adicionadas por ti. Como é óbvio, as linhas que marcam o conflito ( `<<<< HEAD`, `======` e `>>>>>origin/caria`) têm que ser removidas, não fazem parte do código deste ficheiro.